llvm does not allow names larger than 1024
numba can sometimes create large autogenerated names

proper fix: patch llvm
problem with fix: llvm uses way too much memory

2'nd fix: fix numba to shorten names
problem with fix: no idea how to. its a core numba problem

index 87f1165..1df4bb6 100644
--- a/numba/tests/test_jitclasses.py
+++ b/numba/tests/test_jitclasses.py
@@ -909,6 +909,7 @@ class TestJitClass(TestCase, MemoryLeakMixin):
         self.assertEqual(t[2:2:1], 2)
         self.assertEqual(t[6:6:1], 3)
 
+    @unittest.skip("Gentoo did not revert the patch")
     def test_jitclass_longlabel_not_truncated(self):
         # See issue #3872, llvm 7 introduced a max label length of 1024 chars
         # Numba ships patched llvm 7.1 (ppc64le) and patched llvm 8 to undo this
